.lando.base.yml

```diff
@@ -31,8 +40,6 @@ services:
     hogfrom:
       - appserver
 tooling:
-  drupal:
```

Comment:
I think drupal shell is dead now isn't it? I don't think we use it anymore in any case